### ISSUE TITLE
pyNN ecosystem

### ISSUE THREAD

**Issue by laurentperrinet** [SpikeAI/2020-11_brainhack_Project7#4]
More about pyNN and simulations of SNNs:

* PyNN (pronounced 'pine') is a simulator-independent language for building neuronal network models. : http://neuralensemble.org/PyNN/ 
* [doc](http://neuralensemble.org/docs/PyNN/) and [examples](http://neuralensemble.org/docs/PyNN/examples.html),
* once connected to https://spinn-20.cs.man.ac.uk/ read and run the `SynfireExample.ipynb` notebook

 * [neo](https://github.com/NeuralEnsemble/python-neo)

leave comments if you have questions...

**Comment by laurentperrinet** [comment]
you can see this [example](https://github.com/SpikeAI/2020-11_brainhack_Project7/blob/main/dev/2020-11-08_balancing_a_network_B.ipynb) which is close (in terms of number of neurons, simulation time, ...) to the type of SNN we wich to model...

**Comment by laurentperrinet** [comment]
can you all run the `SynfireExample.ipynb` notebook ?

**Comment by dabane-ghassan** [comment]
> can you all run the `SynfireExample.ipynb` notebook ?

Yes i was able to run it and produce the final plot, now i'm catching up on balancing_a_network_B.ipynb

**Comment by AntoineGrimaldi** [comment]
Yep I was able to do it too. And to keep it clear, now the idea would be to include tonic in the pyNN framework for the 'stimulus' population right?

**Comment by AntoineGrimaldi** [comment]
> can you all run the `SynfireExample.ipynb` notebook ?

Yep I was able to do it too. And to keep it clear, now the idea would be to include tonic in the pyNN framework for the 'stimulus' population right?

**Comment by laurentperrinet** [comment]
yes - I should try to push a notebook with our current solution...
